templates/workloads/partials/cf/sidecars.yml

```diff
@@ -49,6 +49,10 @@
   DockerLabels:{{range $name, $value := $sidecar.DockerLabels}}
     {{$name | printf "%q"}}: {{$value | printf "%q"}}{{end}}
 {{- end -}}
+{{- if $sidecar.Dependencies}}
```

Comment:
It looks like the CloudFormation template for container dependencies documented [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html#cfn-ecs-taskdefinition-containerdefinition-dependson) uses the `DependsOn` key and requires a list of [`ContainerDependency` objects](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdependency.html), rather than a map. Can we update this and the WorkloadContainer CF templates to match the cloudformation spec?

Reply:
Adjusted to better match the DependsOn / Conditions / ContainerName format.